Mobile Buy SDK/Mobile Buy SDK/Operations/BUYCheckoutOperation.m

```diff
@@ -29,15 +29,15 @@
 #import "BUYClient+Internal.h"
 #import "BUYPaymentToken.h"
 #import "BUYRequestOperation.h"
+#import "BUYStatusOperation.h"
 
 @interface BUYCheckoutOperation ()
 
+@property (strong, nonatomic, readonly) BUYClient *client;
```

Comment:
Since this works without problems in practise, and has no functional risks that we can see, we can leave it for now.